src/spikeinterface/preprocessing/whiten.py

```diff
@@ -55,6 +62,8 @@ def __init__(
         recording,
         dtype=None,
         apply_mean=False,
+        regularize=False,
+        regularize_kwargs={},
```

Comment:
```suggestion
        regularize_kwargs=None,
```
And in the init use:
```
regularize_kwargs = regularize_kwargs is regularize_kwargs is not None else {}
```